vic/extensions/rout_stub/rout.mk

```diff
@@ -0,0 +1,8 @@
+# make a list of all *.h files in routing folder
+INCL_ROUT := $(wildcard ${EXTPATH}/${ROUT}/include/*.h)
```

Comment:
This is great.  I think we should use this same logic in the driver makefiles as well. That should go in a separate branch/PR.